internal/pkg/util/fs/files/action_script.sh

```diff
@@ -1,9 +1,9 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
@cclerget is this interpreted by the apptainer built-in shell interpreter?

Reply:
no this is comment for the interpreter